app/_how-tos/get-started-with-metering-and-billing.md

```diff
@@ -0,0 +1,253 @@
+---
+title: Get started with Metering and Billing in {{site.konnect_short_name}}
+description: Learn how to...
+content_type: how_to
+
+permalink: /metering-and-billing/get-started/
+breadcrumbs:
+  - /metering-and-billing/
+
+products:
+    - gateway
+    - metering-and-billing
+
+works_on:
+    - konnect
+
+tags:
+    - get-started
+
+tldr: 
+  q: What is Metering and Billing in {{site.konnect_short_name}}, and how can I get started with it?
+  a: |
+    blah
+
+tools:
+    - deck
+  
+prereqs:
+  skip_product: true
+  inline:
+    - title: cURL
+      content: |
+        [cURL](https://curl.se/) is used to send requests to {{site.base_gateway}}. 
+        `curl` is pre-installed on most systems.
+  entities:
+    services:
+        - example-service
+    routes:
+        - example-route
+
+cleanup:
+  inline:
+    - title: Clean up Konnect environment
+      include_content: cleanup/platform/konnect
+      icon_url: /assets/icons/gateway.svg
+
+min_version:
+    gateway: '3.4'
+next_steps:
+  - text: See all {{site.base_gateway}} tutorials
+    url: /how-to/?products=gateway
+  - text: Learn about {{site.base_gateway}} entities
+    url: /gateway/entities/
+  - text: Learn about how {{site.base_gateway}} is configured
+    url: /gateway/configuration/
+  - text: See all {{site.base_gateway}} plugins
+    url: /plugins/
+automated_tests: false
+---
+
+## Create a Consumer
+
+Create the two consumers, one for the free and one for premium. These will be mapped to our customers/subjects later on
+
+[Consumers](/gateway/entities/consumer/) let you identify the client that's interacting with {{site.base_gateway}}.
+We're going to use key [authentication](/gateway/authentication/) in this tutorial, so the Consumer needs an API key to access any {{site.base_gateway}} Services.
+
+<!--vale off-->
+{% entity_examples %}
+entities:
+  consumers:
+    - username: kong-air
+      keyauth_credentials:
+        - key: air-key
+    - username: kong-travel
+      keyauth_credentials:
+        - key: travel-key
+{% endentity_examples %}
+<!--vale on-->
+
+## Enable authentication
+
+Authentication lets you identify a Consumer so that you can apply rate limiting.
```

Comment:
Authentication lets you identify a Consumer so that you can apply rate limiting. =>
Authentication lets you identify a Consumer so you can invoice Customers after their consumption.